### PR TITLE
Ebpf fallback

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

**Comment by safchain** [comment]
Tested with success with kernel version 4.15

**Comment by safchain** [comment]
@andrekassis @eranra can you confirm that works for you ?

**Comment by eranra** [comment]
@safchain, @andrekassis is on this :-)

**Comment by andrekassis** [comment]
working;)

File: flow/probes/ebpf.go [hunk 1]
```diff
@@ -603,26 +603,35 @@ func LoadJumpMap(module *elf.Module) error {
...
 		// split to skip to kernel stack trace
 		errs := strings.Split(err.Error(), ":")
 		if len(errs) > 1 {
 			logging.GetLogger().Debugf("eBPF kernel stacktrace: %s", errs[1])
 		}
+		logging.GetLogger().Errorf("Unable to load eBPF elf binary (host %s) from bindata: %s, trying to fallback", runtime.GOARCH, errs[0])
+
+		module, err = loadModuleFromAsset("probe/ebpf/flow.o")
+		if err != nil {
+			return nil, fmt.Errorf("Unable to load eBPF elf binary (host %s) from bindata: %s", runtime.GOARCH, errs[0])
```
Comment (hunchback) [inline flow/probes/ebpf.go]: the errs[0] is the value calculated at L622 while you need to rather extract error from L628.

Generally i would add a helper func handling loadModuleFromAsset() plus errors.

Comment (safchain) [reply flow/probes/ebpf.go]: I remove the errs thing has not useful and error prone. I kept the log outside of the load function as they are differents.

**Comment by safchain** [comment]
rerun skydive-functional-tests-backend-orientdb

**Pull request merged**